### ISSUE TITLE
Suggestion for the make_covariates function

### ISSUE THREAD

**Issue by agruss2** [James-Thorson-NOAA/VAST#386]
Just an idea for future development/enhancement:
The nearest neighbour procedure in the make_covariates function is not ideal for stream network models. So, in the future, it may be good to adopt an interpolation procedure that is more suited to stream networks in the make_covariates function when the VAST model of interest is a stream network model. 
I reckon this issue may be easier to address with tinyVAST than with VAST?

**Comment by James-Thorson** [comment]
Yeah I doubt I'll do that level of development in VAST. But I'm open to
ideas about developing helper functions for tinyVAST, perhaps in an
optional tinyVASTextras package
